kolibri/plugins/learn/assets/src/composables/useProgressTracking.js

```diff
@@ -231,6 +231,7 @@ export default function useProgressTracking(store) {
       }
       if (response.data.complete) {
         set(complete, true);
+        set(progress_state, 1);
```

Comment:
Make sure to update the frontend progress_state to 1 if backend rounding has declared this resource complete.